misk/src/main/kotlin/misk/security/authz/fake/FakeCallerAuthenticator.kt

```diff
@@ -29,3 +33,8 @@ class FakeCallerAuthenticator : MiskCallerAuthenticator {
   }
 }
 
+data class FakeAccess(
```

Comment:
Maybe this should just be an instance of MiskCaller ?

Reply:
Good suggestion! Fixed!